js/utils/autoUpdate.js

```diff
@@ -0,0 +1,60 @@
+import $ from 'jquery';
+import { ipcRenderer } from 'electron';
+import app from '../app';
+import Dialog from '../views/modals/Dialog';
+
+let statusMsg;
+let removeStatusMsgTimout;
+
+export function showUpdateStatus(status = '', msg = '', type = 'message') {
```

Comment:
Earlier code passed in the msg variables from various updater events, I took that out and forgot to clean up here. Good catch.